Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -624,6 +627,18 @@ shows the resulting mesh.
 View of 3D meshes produced from a polyhedral domain. (i) is a view of file out_1.mesh and (ii) is a view of file out_2.mesh. Code from subsection \ref Mesh_3_subsection_examples_polyhedral generates these files."
 \cgalFigureEnd
 
+\subsubsection Mesh_3RemeshingPolyhedralSurface Remeshing a Polyhedral Surface
+
+The following code creates a polyhedral domain, with only one polyhedron,
+and no "bounding polyhedron", so the volumetric part of the domain will be empty.
+This allows to remesh a surface, and is equivalent to the function `make_surface_mesh()`. 
+
```

Comment:
The polyhedral domain is created using an undocumented constructor. I wonder if we should document it.

That is only a few lines of documentation.